media/com_fabrik/js/tipsBootStrapMock.js

```diff
@@ -230,6 +244,13 @@ define(['jquery', 'fab/fabrik'], function (jQuery, Fabrik) {
                         .css(tp)
                         .addClass(placement)
                         .addClass('in');
+                    if ( (tippos.includes("top") || tippos.includes("bottom")) 
+                        && xpos > 0 && xpos > leftpos+15 && xpos < leftpos+maxwidth-15 ) {
+                        $tip.find("div.arrow").css('left',xpos-leftpos);
+                    }                           
+                    if(tippos.includes("top") && $tip[0].offsetHeight !== actualHeight){                       
```

Comment:
If tip position is 'top' and popover width had been changed, then adjust the top position to accommodate the new popover height.